esmvaltool/utils/recipe_test_workflow/doc/source/adding_a_recipe.rst

```diff
@@ -0,0 +1,99 @@
+   should be added to the ``medium`` option. When adding the recipe it should
+   follow the format of ``recipe_new_recipe \,``, unless it is the last one in
+   the list, in which case it should follow the format of ``recipe_new_recipe``.
+
+#. If the duration of the recipe is larger than the value specified by the
+   ``execution time limit`` option in the ``[[COMPUTE]]`` section in the
+   aforementioned site-specific ``.cylc`` file, and / or the memory usage of
+   the recipe is larger than the value specified by the ``--mem`` option in the
+   ``[[[directives]]]`` section in the ``[[COMPUTE]]`` section, add a section
+   (in alphabetical order) to this file as shown below (round the duration to
+   the nearest second).
+
+   Supplementary to the paragraph above, we advise execution with a duration of
+   less than ~1m45s should use an execution time limit of PT2M, and round up to
+   the nearest integer for the memory usage.::
+
+    [[process<fast=recipe_albedolandcover>]]
+    # Actual: 0m31s, 2.5 GB on 2024-04-08.
+    execution time limit = PT2M
+    [[[directives]]]
+        --mem = 3G
+
+
+   **Please note**: When you add the your modified version of the above example
+   to your site-specific ``.cylc`` file, the key *fast* should match the
+   category you placed your recipe within the ``[task parameters]`` section
+   of the flow.cylc
+
+#. Stop any running ``recipe_test_workflow`` workflows::
+
+    cylc stop recipe_test_workflow
+
+#. Run the |RTW|; it is expected that the ``compare`` task will fail::
+
+    cylc vip -O <site>
+
+#. Update the Known Good Outputs (|KGOs|):
+
+   #. Locate the workflow run folder of the workflow you just completed.
+
+   #. Recursively copy the recipe output directory (i.e.
+      ``recipe_<recipe>_<date>_<time>``) from the
+      ``${HOME}/cylc-run/recipe_test_workflow/run1/share/cycle/<cycle>``
+      directory to your site-specific KGO directory, as detailed by the
+      ``KGO_ROOT_PATH`` option in the site-specific ``.conf`` file in the
+      ``esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/opt/``
+      directory::
+
+      ``cp -r <directory_of_recipe_output_cycle_folder> <KGO_rootpath_folder>``
+
+   #. Change directory to the rootpath KGO directory::
+
+      ``cd <KGO_rootpath_directory>``
+
+   #. Allow write permissions for all users on the directory and it's
+      subdirectories of the recipe you've added to the KGO folder::
+
+      ``chmod a+w <the_directory_of_the_recipe_you_have_copied_into_the_KGO_folder>``
+
+#. Stop any running ``recipe_test_workflow`` workflows::
+
+    cylc stop recipe_test_workflow
+
+#. Run the |RTW| again; the ``compare`` task should now succeed::
+
+    cylc vip -O <site>
+
+#. Add the recipe to the documentation; add a link to the recipe to the list of
+   "Currently tested recipes" in ``doc/source/tested_recipes.rst`` in
+   alphabetical order.
+
+#. Commit and push your changes.
```

Comment:
... changes, create a PR, and add @alistairsellar and @ehogan as reviewers, see [ESMValTool: PR #3664](https://github.com/ESMValGroup/ESMValToolpull/3664) for an example.

Hmmm, although I wonder whether we should add us to the [CODEOWNERS file](https://github.com/ESMValGroup/ESMValTool/blob/main/.github/CODEOWNERS). @alistairsellar; any thoughts on this? 😊

Reply:
> I wonder whether we should add us to the [CODEOWNERS file](https://github.com/ESMValGroup/ESMValTool/blob/main/.github/CODEOWNERS). @alistairsellar; any thoughts on this? 😊

I think that's a good idea @ehogan. I think we would then automatically get added as reviewers?

Reply:
Yes, that's correct 😊 I have opened a new issue to do this (#3667) and adjusted the suggestion above 👍

Reply:
I'm presuming no changes are necessary here as issue #3667 has been completed

Reply:
The change I suggested in https://github.com/ESMValGroup/ESMValTool/pull/3614#discussion_r1647394733 was edited to remove adding me and Alistair as reviewers (since this is now done via the `CODEOWNERS` file), so the latest version of the suggestion still needs implementing 👍

Reply:
Hopefully this is now done in commit 63b93a1d8

Reply:
I've made amendments to the names of these links to the previous naming in commit 50a6eccc1

Reply:
Final changes made in 433e0b696